### DISCUSSION TITLE
Development iOS Provisioning Profiles  Yes and is NOT compatible with Titanium

### DISCUSSION THREAD

**Discussion by jmcerrejon** [appcelerator/titanium_mobile#12550] [Q&A] [answered]
When I run` appc info`, I get the next:

```sh
Development iOS Provisioning Profiles
  iOS Team Provisioning Profile: *
...
Managed                   = Yes and is NOT compatible with Titanium
```
Should I worry about that?

**Comment by m1ga** [comment]
https://github.com/appcelerator/titanium_mobile/blob/07592855ee22082c16f25f94155f3c759ba477c5/iphone/cli/lib/info.js#L224
as soon as it is managed it will show that text.
That said: I don't have a managed certificate :smile:

**Reply by jmcerrejon** [reply]
How rude the string! 😂 
Thks Michael!
`/score give`... ups! 😛

**Comment by hansemannn** [comment, marked as answer]
The managed profile appears because it is synced via Xcode and is automatically generated. Titanium currently only handled "manual" profiles, which is an important thing to improve in the future. @cb1kenobi and I had long discussions about that in the past :)

**Reply by jmcerrejon** [reply]
Very valuable info. Thks Hans

**Reply by cb1kenobi** [reply]
I would love to support managed provisioning profiles, however we have a chicken and egg problem. We can't enable them out of the box since the Xcode project is generated and the managed profiles won't be created until the Xcode project is manually opened and Xcode is authenticated with your Apple account.

There's a couple things that we could do:

1. Give the Titanium CLI your Apple account and let it "manage" your provisioning profiles. I got a lot of blow back on this idea as people didn't trust the CLI with their credentials. This would get us one step closer to getting rid of the `--pp-uuid` which doesn't change that often and could either be managed or persisted in a user project file.
2. Create the Xcode project as apart of `ti create` and persist it in the project directory. During build, we would use this project as the basis for the generated Xcode project. One problem is all of the Titanium iOS source code is in apart of the Xcode project. If you change SDKs, then your Xcode project would need to have the iOS files re-syncd. Possible solutions would be to either merge the user Xcode project with the one in the SDK -or- move all of the iOS code into frameworks that bundle up the Titanium iOS source code. The latter would be smart as it would speed up builds and allow entire modules (like media or network) to be excluded from the app (just like Android does).

I'm sure there are more ideas, but I'd be curious what others come up with.
